backend/Dockerfile

```diff
@@ -52,17 +50,20 @@ RUN pip install --no-cache psycopg[binary] gunicorn
 # copy project
 COPY --chown=app:app . $APP_HOME
 
-# change to the app user
-USER app
+# # change to the app user
+# USER app
 
 ARG SECRET_KEY
 ENV SECRET_KEY=${SECRET_KEY}
 ENV DJANGO_SETTINGS_MODULE=backend.production_settings
 
+# RUN python manage.py collectstatic --no-input --clear
```

Comment:
How are static files handled now?
Following this guides:
* https://docs.djangoproject.com/en/4.2/howto/static-files/#configuring-static-files
* https://docs.djangoproject.com/en/4.2/howto/static-files/deployment/#serving-static-files-in-production

It's probably a good idea to serve generated, static files from a different source than the web server.
I don't know how to handle this task though.

Reply:
First off, I'm not sure why I removed this. Maybe because I saw it being run in the demo command. I'll double check what is actually being run. 

As far as offloading serving the static files, I'll echo what I said in another comment, that if we're looking to offload our form work to the frontend, it's not worth our time doing small optimizations for django static files. The average user will never touch a django static file.